execution/executor-benchmark/src/lib.rs

```diff
@@ -103,6 +103,7 @@ pub fn run_benchmark<V>(
     mut transactions_per_sender: usize,
     connected_tx_grps: usize,
     shuffle_connected_txns: bool,
+    maybe_hotspot_probability: Option<f32>,
```

Comment:
I think 'hotspot_probability' name should be good enough. Option<f32> will indicate the maybe

Reply:
fixed